lib/commands/release/index.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
+import type * as BalenaSdk from 'balena-sdk';
+import jsyaml = require('js-yaml');
+import { getRelease } from '../../utils/release-utils';
+
+interface FlagsDef {
+	help: void;
+	composition?: boolean;
+}
+
+interface ArgsDef {
+	commitOrId?: string;
+}
+
+export default class ReleaseCmd extends Command {
+	public static description = stripIndent`
+		Get info for a release.
+`;
+	public static examples = ['$ balena release commit', '$ balena release id'];
```

Comment:
The `balena device` example though include valid example values
See https://github.com/balena-io/balena-cli/blob/88fc3f77146144eb56b906aa413ef2f8e6b84c95/lib/commands/device/index.ts#L62